tests/security/test_sql_injection.py

```
@@ -0,0 +1,325 @@
+            'delete_account', 'admin_delete_account', 'get_account_coins',
+            'add_coins_to_account', 'get_user_accounts_with_coins'
+        ]
+        
+        for func_name, func in functions:
+            if func_name in sql_functions:
+                # Get the source code
+                source = inspect.getsource(func)
+                
+                # Check for dangerous patterns
+                dangerous_patterns = [
+                    f'f"',  # f-string in SQL
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)